#
# typo_dnspark.patch send by Marco.
# Applied on ddclient Rev 72
#
--- ddclient.orig	2007-07-31 15:44:20.000000000 +0100
+++ ddclient	2007-07-31 15:45:52.000000000 +0100
@@ -2875,7 +2875,7 @@

 The 'dnspark' protocol is used by DNS service offered by www.dnspark.com.

-Configuration variables applicable to the 'easydns' protocol are:
+Configuration variables applicable to the 'dnspark' protocol are:
   protocol=dnspark             ##
   server=fqdn.of.service       ## defaults to www.dnspark.com
   backupmx=no|yes              ## indicates that DNSPark should be the secondary MX
